Source/WebCore/Modules/mediastream/RTCEncodedFrame.cpp

```diff
@@ -50,6 +50,13 @@ void RTCEncodedFrame::setData(JSC::ArrayBuffer& buffer)
     m_data = &buffer;
 }
 
+uint64_t RTCEncodedFrame::timestamp() const
+{
+    if (!m_timestamp)
+        m_timestamp = m_frame->timestamp();
```

Comment:
do we need to cache it  in RTCEncodedFrame? why not read m_frame directly >

Reply:
We need to cache it since m_frame is nullified when the frame gets written.
Maybe we should initialise m_timestamp at constructor time.